thin-replica-server/include/thin-replica-server/thin_replica_impl.hpp

```diff
@@ -77,13 +80,15 @@ struct ThinReplicaServerConfig {
                           SubBufferList& subscriber_list_,
                           std::unordered_set<std::string>& client_id_set_,
                           const uint16_t update_metrics_aggregator_thresh_ = 100,
+                          const bool use_unified_certs = false,
```

Comment:
Please remove the 'const' attribute - it does not make sense when you pass a variable 'by value'. And don't specify a default value for use_unified_certs - error-prone approach. Don't take as an example the code written here :-)